modules/ROOT/pages/secure-configuration-properties.adoc

```diff
@@ -241,112 +352,18 @@ When using encrypted properties, it is especially important to **secure access t
 * ECB
 * OFB
 
-== Using the Extension in Anypoint Studio 7
-
-You can use this extension by adding it as a dependency in your Mule app.
-
-=== Installing the Extension
-
-. Open your Mule project in Anypoint Studio.
-
-. Go to the Mule Palette.
-
-. Select *Search in Exchange*, and search for the *Mule Secure Configuration Property Extension*.
-
-. Add the extension.
-
-=== Adding Secure Configuration Properties to your App
-
-. Go to your Mule app configuration file.
-
-. Select the `Global Elements` tab.
-
-. Click the `Create` button.
-
-. From the search bar, select `Secure Properties Config`.
-
-. Configure the global element with a `File` location, `Key`, `Algorithm`, and `Mode`:
-
-image::secure-configuration-properties-studio.png[config extension]
-
-[[secure_props_tool]]
-== Secure Properties Tool
-
-You can download the https://docs.mulesoft.com/downloads/mule-runtime/4.2/secure-properties-tool.jar[JAR file] for this tool so you can encrypt or decrypt single values, and complete files (both, YAML and Properties files).
-This tool supports three encryption modes: `string`, `file`, and `file-level`
-
-=== Encrypt or Decrypt Single values
-
-Use the `string` mode to encrypt or decrypt a single value provided as a parameter.
 
-[source,console.linenums]
-----
-java -jar secure-properties-tool.jar string <encrypt|decrypt> <algorithm> <mode> <key> <value>
-----
+== Frequently Asked Questions
 
-Replace `<value>` with any desired string value you want to process, for example:
-
-[source,console.linenums]
-----
-java -jar secure-properties-tool.jar string encrypt Blowfish CBC mulesoft "some value to encrypt"
-----
+*What if a property is defined in multiple files?*
 
-=== Encrypt or Decrypt Files
+In that case, the actual property's value will be the one in which is first defined.
 
-In addition to encrypting strings, the encryption tool is able to encrypt complete `.yaml` or `.properties` files.
+*What if I want to define a secure property to depend on a previously defined secure property?*
```

Comment:
```suggestion
*Can I define a secure property that depends on a previously defined secure property?*
```

Reply:
Or do we mean, "Can I create a new secure property that is based on a previous one?" or "...that depends on a previous one?"